<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View3DTVCalcuMatDataWidget</class>
 <widget class="QWidget" name="View3DTVCalcuMatDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>832</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>铁路隧道建筑限界测量系统-双目计算三维结果(.mdat)查看工具</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/image/icon.png</normaloff>:/image/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>单隧道综合及图形修正</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button4">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>
                    &lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
                    &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
                    p, li { white-space: pre-wrap; }
                    &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
                    &lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;申请编辑单隧道&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                  </string>
        </property>
        <property name="styleSheet">
         <string notr="true">
                    QPushButton{
                    border-width:4px;
                    border-image:url(:image/tabbutton1_up.png)
                    4 4 4 4 stretch stretch;
                    }

                    QPushButton:hover{
                    border: 0.1px solid;
                    border-image:url(:image/tabbutton1.png)
                    }

                    QPushButton:pressed{
                    border: 0.1px solid;
                    border-image:url(:image/tabbutton2.png)
                    }
                  </string>
        </property>
        <property name="text">
         <string>申请编辑单隧道</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">image: url(:/image/arrow.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>
                    &lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
                    &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
                    p, li { white-space: pre-wrap; }
                    &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
                    &lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;单隧道综合&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                  </string>
        </property>
        <property name="styleSheet">
         <string notr="true">
                    QPushButton{
                    border-width:4px;
                    border-image:url(:image/tabbutton1_up.png)
                    4 4 4 4 stretch stretch;
                    }

                    QPushButton:hover{
                    border: 0.1px solid;
                    border-image:url(:image/tabbutton1.png)
                    }

                    QPushButton:pressed{
                    border: 0.1px solid;
                    border-image:url(:image/tabbutton2.png)
                    }
                  </string>
        </property>
        <property name="text">
         <string>单隧道综合</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">
                    QPushButton{
                    font: 87 italic 8pt &quot;Arial Black&quot;;
                    font: 75 italic 9pt &quot;Agency FB&quot;;
                    border-width:4px;
                    border-image: url(:/image/exit.png)
                    2 2 2 2 stretch stretch;
                    }

                  </string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page4"/>
     <widget class="QWidget" name="page5"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
